src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <title>Javascript Group Capstone Project By David Yannick and Saningo  </title>
+</head>
+<body>
+  <header>
+    <div class="navbar">
+      <a class="link-logo" href="./index.html"> <img class="logo" src="../src/img/logo.png" alt="logo"></a>
+      <nav class="links">
+        <ul class="link-lists">
+          <li class="link-list"><a href="./index.html" class="link tv-show-link">.</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
```

Comment:
@ShoiraTa you might as well consider asking how much of my contribution (commits) led to the completion of the project before drawing conclusions.

Reply:
Hi @LYANGEND , thanks so much for leaving a comment, now, since you mentioned your contribution I have no doubts that you indeed worked and contributed a lot. But how could I know before? 🙈 
I usually check the `insights` section to see the contributions of team members. But you can see that based on statistics, you had no commits: 
 <img width="1283" alt="Screenshot 2022-04-22 at 18 14 29" src="https://user-images.githubusercontent.com/77038610/164732236-73f2ba2b-a187-47a9-aba6-efa2be2e9b77.png">

To avoid this kind of misunderstandings, in the future try to make commits and keep an eye on `insights` section, especially during the final project. 

One more time, sorry if I somehow misinterpreted the situation I was trying to follow the microverse guideline on reviewing projects 🙏🏻